README.md

```diff
@@ -77,6 +78,20 @@ mvn verify -f acceptance-tests/pom.xml -Pacceptance-test
 See [Automated Acceptance Tests](#automated-acceptance-tests) section for more details on how to further optimise the
 development cycle when working on acceptance tests.
 
+## Auto-export
+
+When making changes to the CMS or console data, you may want your changes to be persisted in the yaml configuration 
+to be boot strapped into the application on startup. The easiest way to do this is to turn auto-export on in the CMS
+(gray button on bottom left of every CMS page). Make sure you turn it on before making any of the changes and turn it
```

Comment:
..every CMS page) or in the Console (top menu options).

Reply:
Nice, didn't know that!